returnn/compile.py

```diff
@@ -2,62 +2,60 @@
 
 from sisyphus import *
 
-Path = setup_path(__package__)
-
 import copy
 import logging
-import os
 import shutil
 import subprocess as sp
-from typing import Optional, Sequence
+from typing import Any, Dict, Optional, Sequence, Union
 
 import i6_core.util as util
 
 from .config import ReturnnConfig
 from .training import PtCheckpoint
 
+Path = setup_path(__package__)
+
 
 class CompileTFGraphJob(Job):
     """
     This Job is a wrapper around the RETURNN tool compile_tf_graph.py
-
     """
 
     __sis_hash_exclude__ = {"device": None, "epoch": None, "rec_step_by_step": None, "rec_json_info": False}
 
     def __init__(
         self,
-        returnn_config,
-        train=0,
-        eval=0,
-        search=0,
-        epoch=None,
-        verbosity=4,
-        device=None,
+        returnn_config: Union[ReturnnConfig, tk.Path, str],
+        train: int = 0,
+        eval: int = 0,
+        search: int = 0,
+        epoch: Optional[Union[int, tk.Variable]] = None,
+        verbosity: int = 4,
+        device: Optional[str] = None,
         summaries_tensor_name=None,
```

Comment:
```suggestion
        summaries_tensor_name: str =None,
```
I think this should be str?

Reply:
I guess so, but I didn't see any example of this.

I'll just set it as string :woman_shrugging: